packages/@jsii/benchmarks/lib/constants.ts

```diff
@@ -0,0 +1,9 @@
+import * as path from 'path';
+
+export const fixturesDir = path.join(__dirname, '..', 'fixtures');
```

Comment:
For some weird reason I prefer to use `resolve` for those (feel free to ignore me)

```suggestion
export const fixturesDir = path.resolve(__dirname, '..', 'fixtures');
```